setup.cfg

```diff
@@ -60,24 +60,20 @@ where=src
 
 [options.extras_require]
 test =
+    pandas; python_version>"3.4"
     pytest
     pytest-cov
-    pandas; python_version>"3.4"
     tabulate
 dev =
-    pytest
+    check-manifest>=0.42
     pandas; python_version>"3.4"
+    pytest
     tabulate
-    check-manifest>=0.42
-    black==20.8b1
-    mypy==0.790
```

Comment:
It's invalid to run this by hand, we should only support running from pre-commit; otherwise we'd have to maintain these with correct versions, etc.

Reply:
I didn't write this ;-).

Reply:
Didn't imply you did. :) I was originally trying to support both, but obviously pre-commit was (automatically) maintained and this one was forgotten.